src/Components/Preview.lua

```diff
@@ -20,6 +21,8 @@ function Preview:init()
 
 	self.monkeyGlobalTable = {}
 
+	self.screenGui = Instance.new('ScreenGui')

```

Comment:
1. This is a non-obvious variable name, try to come up with one that better describes its purpose.
2. Double quotes instead of single quotes.